pkg/storage/storage.go

```diff
@@ -11,7 +11,6 @@ import (
 
 const (
 	S3StorageDriverName = "s3"
-	DefaultGCDelay      = 1 * time.Hour
```

Comment:
Can we get away without updating these imports? If the metadata store is not imported to be used in the storecontroller we should be fine right?